x-pack/plugins/triggers_actions_ui/public/application/components/builtin_alert_types/threshold/expression.tsx

```diff
@@ -125,9 +126,11 @@ export const IndexThresholdAlertTypeExpression: React.FunctionComponent<IndexThr
   );
 
   const getIndexPatterns = async () => {
-    const indexPatternObjects = await loadIndexPatterns();
-    const titles = indexPatternObjects.map((indexPattern: any) => indexPattern.attributes.title);
-    setIndexPatterns(titles);
+    if (getSavedObjectsClient()) {
```

Comment:
It seems this workaround hides index patterns from the search results when using the flyout outside of the management app (ex from Uptime). The original bug seems to work ok without this if statement, maybe we create a separate issue to fix this?